ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/NetListeningTest.java

```diff
@@ -69,7 +69,7 @@ public void shouldReturnFalseWhenNetworkIsNotListening() {
   @Test
   public void getPermissions() {
     List<String> permissions = method.getPermissions();
-    assertThat(permissions).containsExactlyInAnyOrder("net:*", "net:listening", "*:*");
+    assertThat(permissions).containsExactly("*:*", "net:*", "net:listening");
```

Comment:
What's the reason for this change?

Reply:
it just made sense to have the items in order of granularity from widest to narrowest

Reply:
also if they are tested in that order then the log.debug messages will make sense